src/host/ut_host/SelectionTests.cpp

```diff
@@ -378,27 +383,27 @@ class SelectionTests
 
         // start from position 10 before end of row (80 length row)
         // row is 2
-        // selection is 10 characters long
+        // selection is 9 characters long

```

Comment:
uh? wut

Reply:
This whole area was a bit weird to work with.
1. the test text has a Trailing byte at 0 (which should not be happening)
2. the code used to operate by identifying the starting point of a selection and expanding it by X _characters_. We moved away from this and, instead, just create a RECT that expands its left and right side by _cells_.

Since a wide glyph (presumably being one character) can hold two cells, we had to rewrite some of this a little bit.

Reply:
I manually did some tests though (and added some for Michael below). So this should be working fine.